boilerplate/README.md

```diff
@@ -0,0 +1,3 @@
+# Hacked Boilerplate
```

Comment:
Not to add more work, but I think these relevant portions of boilerplate would add value to other repos. Can we move these relevant files into boilerplate? We have other workloads like scanning and suricata repos.

Reply:
Going to close this PR for now and work on that first, to revisit this afterwards